ipatests/test_integration/test_epn.py

```diff
@@ -224,7 +224,10 @@ def test_EPN_config_file(self):
         """
         epn_conf = "/etc/ipa/epn.conf"
         epn_template = "/etc/ipa/epn/expire_msg.template"
-        cmd1 = self.master.run_command(["rpm", "-qc", "freeipa-client-epn"])
+        if tasks.get_platform(self.master) != "fedora":
+            cmd1 = self.master.run_command(["rpm", "-qc", "ipa-client-epn"])
+        else:
+            cmd1 = self.master.run_command(["rpm", "-qc", "freeipa-client-epn"])
```

Comment:
@miskopo @flo-renaud 
What you think about `*ipa-client-epn` instead of if-else?
centos and debian system uses `freeipa-*` prefix too.

Reply:
Sadly, `rpm` command doesn't like glob expressions (tried that, but maybe I missed something). If it was `dnf`, it would work, byt not with `rpm`. Regarding debian, this test is skipped entirely.

Reply:
I checked Centos and it uses `ipa-*` prefix, so currently it should work as it is.